automation_tools/__init__.py

```diff
@@ -393,12 +393,14 @@ def setup_abrt():
     packages = [
         'abrt-cli',
         'rubygem-smart_proxy_abrt',
-        'rubygem-smart_proxy_pulp'
+        'rubygem-smart_proxy_pulp',
+        'ruby193-rubygem-foreman_abrt'
     ]
     for package in packages:
         run('yum install -y {0}'.format(package))
 
     run('systemctl restart foreman')
+    run('touch /usr/share/foreman/tmp/restart.txt')
```

Comment:
`touch` is either to create empty file or to change the timestamps of file..forman is already restarting at line 402